fuel-tx/src/transaction/fee.rs

```diff
@@ -159,68 +171,87 @@ mod tests {
         .with_gas_per_byte(2)
         .with_gas_price_factor(3);
 
+    fn gas_to_fee(gas: u64, gas_price: Word) -> f64 {
+        let fee = gas * gas_price;
+        fee as f64 / PARAMS.gas_price_factor as f64
+    }
+
     #[test]
     fn base_fee_is_calculated_correctly() {
```

Comment:
Might be worth considering adding multi-sig tests?

Reply:
I added additional [tests for multiple signed inputs](https://github.com/FuelLabs/fuel-vm/pull/613/files#diff-a9aa3c1302258ae48cda7b9cb2998209f3b240a4bc9f64e58d221d97dd2488d1R908). Is that what you mean by multi-sig tests?